inference_rules.adoc

```diff
@@ -135,24 +135,15 @@ described in the table below.
 
 |===
 |Scenario |Query Generation |Duration |Samples/query |Latency Constraint |Tail Latency | Performance Metric
-|Single stream |LoadGen sends next query as soon as SUT completes the previous query | 1024 queries and 600 seconds |1 |None |90% | 90%-ile measured latency
-|Server |LoadGen sends new queries to the SUT according to a Poisson distribution |270,336 queries and 600 seconds |1 |Benchmark specific |99% | Maximum Poisson throughput parameter supported
+|Single stream |LoadGen sends next query as soon as SUT completes the previous query | 600 seconds |1 |None |90%* | 90%-ile early-stopping latency estimate
+|Server |LoadGen sends new queries to the SUT according to a Poisson distribution | 600 seconds |1 |Benchmark specific |99%* | Maximum Poisson throughput parameter supported
 |Offline |LoadGen sends all samples to the SUT at start in a single query | 1 query and 600 seconds | At least 24,576 |None |N/A | Measured throughput
-|Multistream | Loadgen sends next query, as soon as SUT completes the previous query | 270,336 queries and 600 seconds | 8 | None | 99% | 99%-ile measured latency|
+|Multistream | Loadgen sends next query, as soon as SUT completes the previous query | 600 seconds | 8 | None | 99%* | 99%-ile early-stopping latency estimate|
 |===
 
-The number of queries is selected to ensure sufficient statistical confidence in
-the reported metric. Specifically, the top line in the following table. Lower
-lines are being evaluated for future versions of MLPerf Inference (e.g., 95%
-tail latency for v0.6 and 99% tail latency for v0.7).
+An early stopping critereon (described in more detail in <<appendix-early_stopping>>) allows for runs with relatively few number of processed queries to be valid, with the penalty that the effective computed percentile will be slightly higher.  This penalty counteracts the increased variance inherent to runs with few queries, where there is a higher probability that a particular run will, by chance, report a lower latency than the system should reliably support.
 
-|===
-|Tail Latency Percentile |Confidence Interval |Margin-of-Error |Inferences |Rounded Inferences
-|90%|99%|0.50%|23,886|3*2^13 = 24,576
-|95%|99%|0.25%|50,425|7*2^13 = 57,344
-|97%|99%|0.15%|85,811|11*2^13 = 90,112
-|99%|99%|0.05%|262,742|33*2^13 = 270,336
-|===
+In the above table, tail latency percentiles with an astrix represent the theretical lower limit of measured percentile for runs processing a very large number of queries.  Submitters may opt to run for longer than the time listed in the "Duration" column, in order to decrease the effect of the early stopping penalty.
```

Comment:
copy edit: "asterisk"

Reply:
Also, the above table is may be somewhat useful in terms of providing guidance for aggressive submitters as to when might be a good time to stop.

Reply:
Fixed copy edit, and re-added the table.